src/core/camera.js

```diff
@@ -624,6 +665,9 @@
      * @param {number} zoom The zoom scale to apply
      */
     this.zoom = function (zoom) {
+      if (zoom === 1) {
+        return;
```

Comment:
It seems silly to do a bunch of needless multiplications when we know we don't have to.  This code isn't particularly expensive, and multiplying by 1 should never result in a precision change, but as a general habit, the less operations we do, the faster things run and the less chance for loss of precision.